### DISCUSSION TITLE
Paragraph limit 1024

### DISCUSSION THREAD

**Discussion by draftgo** [langgenius/dify#3328] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

Paragraph limit 1024

### 2. Describe the feature you'd like to see

Paragraph limit 1024

### 3. How will this feature improve your workflow or experience?

Paragraph limit 1024

### 5. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by crazywoola** [comment]
https://github.com/langgenius/dify/blob/7753ba2d375beb0a5449042fbcbf5ade9a7cffb8/web/app/components/app/configuration/config-var/config-modal/index.tsx#L20

Currently, we have defined a const here which set the limit to 1024.

How much do you want in your case? Can you describe the scenario you have encountered?

**Comment by crazywoola** [comment]
https://github.com/langgenius/dify/pull/3340

**Comment by badbye** [comment]
In my experience, if you just delete the value of `max length` for `paragraph`, keep it empty and save it, it will be unlimited.
I like this "bug"(if it is).
